source/style-guide/style/metadata.txt

```diff
@@ -0,0 +1,19 @@
+.. _metadata:
+
+=============
+Metadata
+=============
```

Comment:
I think the over/under tags here (the lines of equal signs) should be the same length as the word "Metadata"